src/parser.js

```diff
@@ -394,6 +397,33 @@ function stateMachineStatementParser (statement, steps, { isStrict, dialect = 'g
         return;
       }
 
+      if (dialect === 'mysql' && token.value === 'DEFINER') {
```

Comment:
add a comment to briefly talk about `DEFINER`?

Reply:
Yeah, makes sense, added.